social_django/context_processors.py

```diff
@@ -30,7 +31,7 @@ def backends(request):
     """Load Social Auth current user data to context under the key 'backends'.
     Will return the output of social_core.backends.utils.user_backends_data."""
     return {'backends': LazyDict(lambda: user_backends_data(request.user,
-                                                            BACKENDS,
+                                                            settings.AUTHENTICATION_BACKENDS,
```

Comment:
This will also load every backend setup in Django, not just those on python-social-auth.

Reply:
@omab can you explain why you think this would do anything different than the old code? 
 BACKENDS was set to `settings.AUTHENTICATION_BACKENDS` at the top of the file before this change.

Reply:
Similar to the comment below, `user_backends_data` only returns Social Auth backends.

https://github.com/python-social-auth/social-core/blob/master/social_core/backends/utils.py#L60